knora/dsplib/utils/excel_to_json_properties.py

```diff
@@ -70,35 +72,51 @@ def row_to_prop(row: tuple[str, str, str, str, str, str, str, str, str, str, str
     Returns:
         prop (JSON): the property in JSON format
     """
-    name, super_, object_, en, de, fr, it, comment_en, comment_de, comment_fr, comment_it, gui_element, hlist = row
+    name, super_, object_, en, de, fr, it, comment_en, comment_de, comment_fr, comment_it, gui_element, gui_attributes = row
     labels = {}
     if en:
-        labels['en'] = en
+        labels['en'] = en.strip()
```

Comment:
you do lots of `.strip()` all over the place, so you might consider just striping each string when you read the file.

Reply:
That sounds like a good idea, but I don't see an easy solution to do that.